roles/db-create/tasks/main.yml

```diff
@@ -56,15 +56,21 @@
     dest: "{{ swlib_unzip_path }}/dbca_{{ db_name }}.rsp.sh"
     owner: "{{ oracle_user }}"
     group: "{{ oracle_group }}"
-  when: pmon_proc.stdout == "0" and check_oratab.stdout == "0"
+  when:
+    - pmon_proc.stdout == "0"
+    - check_oratab.stdout == "0"
+    - oracle_edition != 'FREE'
```

Comment:
If we have this many logic checks for database edition, might it make sense to create a new playbook and include it conditionally?